WowPacketParser.sln

```diff
@@ -48,68 +48,198 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WowPacketParserModule.V2_4_
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WowPacketParserModule.V8_0_1_27101", "WowPacketParserModule.V8_0_1_27101\WowPacketParserModule.V8_0_1_27101.csproj", "{749EA02D-C727-4697-B979-9ED80FB98085}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WowPacketParserModule.Substructures", "WowPacketParserModule.Substructures\WowPacketParserModule.Substructures.csproj", "{953229B7-01C4-457B-86CC-F0DD1173F217}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
```

Comment:
Fix project and solution files to remove unneccessary x64 and x86 builds, you should only have "Any CPU"